src/index.html

```diff
@@ -17,6 +17,54 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a href="#">Samsung</a>
+          </li>
+          <li>
+            <a href="#">Smartphones</a>
+          </li>
+          <li>
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a href="#">Gadgets</a>
+          </li>
+          <li>
+            <a href="#">Tablets</a>
+          </li>
+          <li>
+            <a href="#">Photo</a>
+          </li>
+          <li>
+            <a href="#">Video</a>
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Use a specific ID or avoid using `#` if it is not intended to be a link.